lang/ar.js

```diff
@@ -39,7 +39,7 @@ export default {
 	"components.input-time-range.endTime": "وقت النهاية",
 	"components.input-time-range.errorBadInput": "يجب أن يكون تاريخ {startLabel} قبل {endLabel}",
 	"components.input-time-range.startTime": "وقت البدء",
-	"components.list-item-drag-handle.default": "إعادة ترتيب الإجراءات",
+	"components.list-item-drag-handle.default": "إعادة ترتيب إجراء المادة لـ {name}",
```

Comment:
before: Rearrange actions
after: Rearrange the material action for {name}
The en term changed so this seems fine